config.py

```diff
@@ -10,9 +10,14 @@
 class Config(object):
     SECRET_KEY = os.environ.get('SECRET_KEY') or \
         'Slava-fakes-it-till-he-makes-it'
+
+    # SQL Alchemy Configs
     SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \
         'sqlite:///' + os.path.join(basedir, 'app.db')
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SQLALCHEMY_ENGINE_OPTIONS = {"pool_pre_ping": True}
```

Comment:
I believe this needs to be mirrored somewhere for the app to actually respond to this. Probably in the `__init__.py` file?

Because just adding this here wouldn't change anything.

Reply:
Why? It configures how SQLAlchemy behaves, not how the app behaves. I didn't need to do anything else after I added it. 

Here's the documentation: 

- https://blog.stigok.com/2021/02/28/sqlalchemy-postgres-ssl-eof-detected.html
- https://docs.sqlalchemy.org/en/13/core/pooling.html#disconnect-handling-pessimistic

Originally it is set up in the engine creation, but I found it to work perfectly fine in `config.py` implementation as well.